deploy.sh

```diff
@@ -29,6 +29,6 @@ sam deploy --stack-name "$stack_name" \
   cri:deployment-source=manual \
   cri:stack-type=localdev \
   --parameter-overrides \
-  Environment=localdev \
+  Environment=dev \
```

Comment:
Glad to see this change! Should we also change the `cri:stack-type`?

Reply:
corrected this so they are both the same